gateway/src/apicast/policy/camel/Readme.md

```diff
@@ -0,0 +1,77 @@
+#  Camel proxy policy
```

Comment:
Not sure if this is the best name for the policy. There's nothing camel-specific, it's valid for any proxy, right?

Reply:
The TLS is only for camel (With termination) and the policy name is suggested to be like that: 
https://issues.redhat.com/browse/THREESCALE-4867